crates/swift-bridge-build/tests/sample-project/src/lib.rs

```diff
@@ -0,0 +1,10 @@
+#[swift_bridge::bridge]
+mod ffi {
+    extern "Rust" {
+        fn hello_rust() -> String;
+    }
+}
+
+fn hello_rust() -> String {
+    String::from("Hello Rust!")
```

Comment:
`"Hello, From Rust!"` just to make it crystal clear that the message came from Rust (as opposed to someone thinking that `Swift` is saying hi to `Rust`.